private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -46,16 +48,25 @@ class Dashboard extends React.Component<Props, State> {
 
   toggleModal = () => {
     this.setState({ isPreviewOpen: !this.state.isPreviewOpen });
-  }
+  };
 
   render() {
+
+    if(this.props.isSearch === true) {
+      return(
+        <DashboardContainer>
+          <SearchPage/>
+        </DashboardContainer>
+      );
+    }
     return (
       <DashboardContainer>
         <Title>Recent</Title>
         <Gallery toggleModal={this.toggleModal}></Gallery>
+        <PreviewModal show={this.state.isPreviewOpen}toggleModal={this.toggleModal}/>
         <Title>Drafts</Title>
         <Gallery toggleModal={this.toggleModal}></Gallery>
-        <PreviewModal show={this.state.isPreviewOpen} toggleModal={this.toggleModal} />
+        <PreviewModal show={this.state.isPreviewOpen} toggleModal={this.toggleModal}/>
```

Comment:
Why is there two preview modals?

Reply:
![image](https://user-images.githubusercontent.com/17441040/74779473-8be15480-526b-11ea-9fff-64bd155fc6af.png)

there are two preview modals, one for each gallery ( Recent and Drafts ). I think we may change this design later on